Create ui-c8y-1018-503-158-about-this-page-content-is-missing-again-for-trusted-certificates-8918.md (#3247)
Cumulocity-IoT/c8y-docs: content/change-logs/platform-services/ui-c8y-1018-503-158-about-this-page-content-is-missing-again-for-trusted-certificates-8918.md

---
date: ""

> title: "About this page" content is missing for Trusted certificates page

Reviewer: ```suggestion
title: Help content on Trusted certificates page is displayed again
```

> title: About this page content is missing again for trusted certificates (#8918)

Reviewer 2: ```suggestion
title: "About this page" content is missing for Trusted certificates page
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63405
version: 1018.503.158
---

> About this page content is missing again for trusted certificates (#8918)

Reviewer 2: ```suggestion
"About this page" content was missing for Trusted certificates page because of outdated link to documetation. New link was provided and help content is visible again.
```

> "About this page" content was missing for Trusted certificates page because of outdated link to documetation. New link was provided and help content is visible again.

Reviewer: ```suggestion
The help content which shows up on clicking the info icon was missing on the **Trusted certificates** as the link to the documentation was outdated. The link has been updated and the help content is visible again.
```
